docs/solana-wallet/api-reference/class.mdx

```diff
@@ -127,12 +129,119 @@ export interface TorusParams {
    * @default false
    */
   useLocalStorage?: boolean;
+  /**
+   * Params to enable whitelabelling of torus website and widget
+   */
+  whiteLabel?: WhiteLabelParams;
 }
-
 ```
 
+### `whiteLabel`
+
 ```typescript
+interface WhiteLabelParams {
+  /**
+   * Whitelabel theme
+   */
+  theme: ThemeParams;
+  /**
+   * Language of whitelabel.
+   *
+   * order of preference: Whitelabel language \> user language (in torus-website) \> browser language
+   */
+  defaultLanguage?: string;
+  /**
+   * Logo Url to be used in light mode (dark logo)
+   */
+  logoDark: string;
+  /**
+   * Logo Url to be used in dark mode (light logo)
+   */
+  logoLight: string;
+  /**
+   * Shows/hides topup option in torus-website/widget.
+   * Defaults to false
+   * @defaultValue false
+   */
+  topupHide?: boolean;
+  /**
+   * Shows/hides billboard in torus-website.
+   * Defaults to false
+   * @defaultValue false
+   */
+  featuredBillboardHide?: boolean;
+  /**
+   * Shows/hides disclaimers on login page. Only works if special logins are hidden
+   * Defaults to false
+   * @defaultValue false
+   */
+  disclaimerHide?: boolean;
+  /**
+   * Language specific link for terms and conditions on torus-website. See (examples/vue-app) to configure
+   */
+  tncLink?: LocaleLinks<string>;
+  /**
+   * Language specific link for privacy policy on torus-website. See (examples/vue-app) to configure
+   */
+  privacyPolicy?: LocaleLinks<string>;
+  /**
+   * Language specific link for privacy policy on torus-website. See (examples/vue-app) to configure
+   */
+  contactLink?: LocaleLinks<string>;
```

Comment:
not supported as params for now, please remove

Reply:
ok, let me remove